src/main/scala/scorex/core/validation/ModifierValidator.scala

```diff
@@ -216,10 +217,16 @@ case class ValidationState[T](result: ValidationResult[T], strategy: ValidationS
 
   /** Shortcut `require`-like method to validate the `id`s are equal. Otherwise returns fatal error
     */
-  def demandEqualIds(given: => Array[Byte], expected: => Array[Byte], fatalError: String): ValidationState[T] = {
+  def demandEqualIds(given: => ModifierId, expected: => ModifierId, fatalError: String): ValidationState[T] = {
     validateEqualIds(given, expected)(d => ModifierValidator.fatal(fatalError, d))
   }
 
+  /** Shortcut `require`-like method to validate the arrays are equal. Otherwise returns fatal error
+    */
+  def demandEqualArrays(given: => Array[Byte], expected: => Array[Byte], fatalError: String): ValidationState[T] = {
+    validateEquals(given)(expected)(d => ModifierValidator.fatal(fatalError, d))
```

Comment:
It's better to use `Arrays.equals` here while `validateEquals ` uses `sameElements`. It could look like:
```
validate(Arrays.equals(given, expected)){ ModifierValidator.fatal(s"$fatalError. Given $given while expected $expected") }
```